src/Common/src/Interop/Windows/mincore/Interop.SetProcessAffinityMask.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.ProcessTopology, CharSet = System.Runtime.InteropServices.CharSet.Unicode, SetLastError = true)]
```

Comment:
Since there's already a using for System.Runtime.InteropServices, do you need to fully qualify CharSet here?  Same comment on other files.